src/D2L.Bmx/BmxConfigProvider.cs

```diff
@@ -15,36 +12,42 @@ internal class BmxConfigProvider( FileIniDataParser parser ) : IBmxConfigProvide
 	public BmxConfig GetConfiguration() {
 		// Main config is at ~/.bmx/config
 		string configFileName = BmxPaths.CONFIG_FILE_NAME;
-
+		var data = new IniData();
+		if( File.Exists( configFileName ) ) {
+			try {
+				var tempdata = parser.ReadFile( configFileName );
+				data.Merge( tempdata );
+			} catch( IniParser.Exceptions.ParsingException ) {
+				//No read permission?
+			}
```

Comment:
I'm curious what permission error you saw.
Also wondering if it makes sense to just catch all errors here and print a warning to the user

Reply:
Yes, catch all errors here make more sense.